src/main/resources/templates/fragments/hour-and-minute-input.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <meta charset="UTF-8" />
+    <title>hour-and-minute-input</title>
+  </head>
+  <body>
+    <div th:fragment="hour-and-minute-input" class="tw-flex tw-items-start tw-gap-3">
+      <span class="tw-flex-1">
+        <th:block
+          th:replace="fragments/input-group :: input-group(~{::body-hours}, ~{::addon-hours}, ${#fields.hasErrors('hours')})"
+        >
+          <th:block th:ref="addon-hours">
+            <th:block th:text="#{hours.abbr}"></th:block>
+          </th:block>
+          <th:block th:ref="body-hours">
+            <input
+              th:field="*{hours}"
+              class="form-control"
+              th:placeholder="#{input.hours.input.placeholder}"
+              type="text"
+              inputmode="numeric"
+              autocomplete="off"
+              data-test-id="overtime-hours"
```

Comment:
unsicher wie geil das ist mit solchen ui Komponenten.

würde das hier erstmal im `overtime` Verzeichnis lassen statt in `fragments`. siehe genau diese `data-test-id=overtime-hours` hier ;-) das betrifft nur Überstunden. gerne dort als separate Datei, damit `overtime-form.html` klein bleibt.

bin mir noch unsicher was man mit dieser template komplexität gewinnt oder verliert.

das fragment
- sagt nichts aus über die benötigten attribute (`xxx.hours`, `xxx.minutes`)
- _muss_ bei Verwendung ein `<form` mit `th:bind=${xxx}>` haben (siehe inputs mit `th:field`)

mit der `input-group` ui komponente kann ich besser leben. die ist super klein und hat keine form element (`input`) die an irgendeinen Kontext gekoppelt werden müssen.